src/translations/indexMapper.js

```diff
@@ -57,7 +57,7 @@ class IndexMapper {
   getPhysicalIndex(visualIndex) {
     const visibleIndexes = this.getNotSkippedIndexes();
     const numberOfVisibleIndexes = visibleIndexes.length;
-    let physicalIndex = null;
+    let physicalIndex = visualIndex;
```

Comment:
This might be a breaking change. Just a note to take a look at it with @wszymanski on Monday

Reply:
Ye, we discussed that earlier. It will be removed.